scripts/gis-material-update/automation/Dockerfile

```
@@ -0,0 +1,35 @@
+FROM python:3.10
+
+ENV TORMAYS_DEPLOYMENT_PROFILE=docker_development
+RUN apt-get update
+
+RUN apt-get install -y software-properties-common wget gdal-bin libgdal-dev gettext-base python3-launchpadlib
+RUN add-apt-repository ppa:rmescandon/yq
+RUN apt install -y yq
+
+WORKDIR /opt
+COPY ./automation/requirements.txt /opt
+RUN python -m venv /opt/venv && /opt/venv/bin/pip install --upgrade pip && /opt/venv/bin/pip install -r requirements.txt
+WORKDIR /haitaton-automation
+COPY ./automation/requirements.txt .
+COPY ./automation/automation.sh .
+WORKDIR /haitaton-gis
+COPY ./config.yaml .
+COPY ./fetch/fetch_all.sh .
+COPY ./fetch/fetch_data.sh .
+COPY ./process/process_data.py .
+COPY ./process/process_data.sh .
+WORKDIR /haitaton-gis/modules
+COPY ./process/modules/ .
+WORKDIR /haitaton-gis-validate-deploy
+COPY ./config.yaml .
+COPY ./validate-deploy/validate_deploy_data.py .
+COPY ./validate-deploy/validate_deploy_data.sh .
+WORKDIR /haitaton-gis-validate-deploy/modules
+COPY ./validate-deploy/modules/ .
+WORKDIR /downloads
+WORKDIR /local_data
+WORKDIR /gis-output
+WORKDIR /haitaton-automation
```

If you specify the directory to copy to, you don't need to change the working directory all the time.
Empty directories can be created with `mkdir`.
```suggestion
COPY ./automation/automation.sh /haitaton-automation/
COPY ./config.yaml /haitaton-gis/
COPY ./config.yaml /haitaton-gis-validate-deploy/
COPY ./fetch/fetch_all.sh /haitaton-gis/
COPY ./fetch/fetch_data.sh /haitaton-gis/
COPY ./process/process_data.py /haitaton-gis/
COPY ./process/process_data.sh /haitaton-gis/
COPY ./process/modules/ /haitaton-gis/modules/
COPY ./validate-deploy/validate_deploy_data.py /haitaton-gis-validate-deploy/
COPY ./validate-deploy/validate_deploy_data.sh /haitaton-gis-validate-deploy/
COPY ./validate-deploy/modules/ /haitaton-gis-validate-deploy/modules/
RUN mkdir -p /downloads /gis-output /local_data
WORKDIR /haitaton-automation
```

---

Change done.